charts/telepresence/README.md

```diff
@@ -101,6 +101,7 @@ The following tables lists the configurable parameters of the Telepresence chart
 | client.routing.allowConflictingSubnets               | Allow the specified subnets to be routed even if they conflict with other routes on the local machine.                      | `[]`                                                                        |
 | client.dns.excludeSuffixes                           | Suffixes for which the client DNS resolver will always fail (or fallback in case of the overriding resolver)                | `[".com", ".io", ".net", ".org", ".ru"]`                                    |
 | client.dns.includeSuffixes                           | Suffixes for which the client DNS resolver will always attempt to do a lookup. Includes have higher priority than excludes. | `[]`                                                                        |
+| argoRollouts.enabled                                 | Enable/Disable the argo-rollouts integration.                                                                               | `false`                                                                     |
```

Comment:
Just a thought: I can imagine a future where multiple `Deployment` alternatives are supported. Might be worth nesting this as `alternateWorkloads.argoRollouts.enabled`.

Reply:
I like this idea, but I don't think it should be "alternative". It should be just "workloads". Let me explain why.

Each workload used today adds overhead. Both to the client-  and cluster-side of Telepresence. If we add an element that defaults to:
```yaml
workloads:
  deployments:
    enabled: true
  replicaSets:
    enabled true
  statefulSets:
    enabled true
  argoRollouts:
    enabled false
```
Then a user that just wants `deployments` to be considered a workload could control this and get rid of the current overhead for replicaSets and statefulSets. This will require some more work on our shared-indexer logic and client code,  and the complete implementation of what I'm suggesting here is beyond the scope of this PR, but adding the `workloads` top level element could be done now as a preparation.

Thoughts?

Reply:
I'm not familiar enough with the codebase to comment on the implementation side, but from a user perspective that sounds great. (especially the "beyond the scope of this PR" bit so I can get Rollout support sooner 😛)